AcademicReward/Views/ShopPage.xaml.cs

```diff
@@ -8,19 +11,42 @@ namespace AcademicReward.Views;
 /// Reviewer: Wil LaLonde
 /// </summary>
 public partial class ShopPage : ContentPage {
+
+    private ShopLogic ShopLogic;
 	public ShopPage() {
 		InitializeComponent();
+        ShopLogic = new ShopLogic();
+        getShopItems();
 	}
 
 	private void Button_Clicked(object sender, EventArgs e) {
-		this.ShowPopup(new AddShopItemPage());
+		this.ShowPopup(new AddShopItemPage(ShopLogic));
+        getShopItems();
 	}
 
-    private void Ticket_Clicked(object sender, EventArgs e) {
-       this.ShowPopup(new ViewShopItemPage());
+    private void SelectedItem(object sender, SelectedItemChangedEventArgs e)
+    {
+        ShopItem selectedItem = e.SelectedItem as ShopItem;
+        if (selectedItem != null)
+        {
+            //ShopItem notificationPopup = new Shopitem();
+            //this.ShowPopup(notificationPopup);
```

Comment:
Can we remove this commented code since this is for the shop and not notifications?

Reply:
Remove commented out code